nservicebus/messaging/immutable-messages.md

```diff
@@ -13,8 +13,19 @@ Usually messages are designed as [DTOs](https://en.wikipedia.org/wiki/Data_trans
 
 Message objects can be made immutable at runtime by:
 
-1. Creating properties with only public getters and initializing these properties via constructor initialization.
-2. Having regular message classes with public getters/setters at the sender, where these classes implement an interface with only public getters. Receivers reference only the interface.
+1. Using record types
+2. Creating properties with only public getters and initializing these properties via constructor initialization.
+3. Having regular message classes with public getters/setters at the sender, where these classes implement an interface with only public getters. Receivers reference only the interface.
+
+
+## Record types
+
+> [!NOTE]
+> Record types are supported starting with NServiceBus 9
```

Comment:
@ramonsmits doing recent docs changes review: Why would records only be supported starting with NSB9? It's just a C# feature. Records were introduced in C# 9 which was .NET 5 so why couldn't you use them in NSB 8 which targets net6.0?

Also, add a period to the end?

Reply:
@bording and I made this modification: https://github.com/Particular/docs.particular.net/commit/7f679839c90823702ba4eb55588c2cc46f7bbacb

Because versions before 7.2 are no longer supported, and the information about records only being supported in NSB9 was wrong. (And we verified that.)